vector/src/main/java/im/vector/webview/DefaultWebViewEventListener.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.webview
+
+import org.matrix.androidsdk.util.Log
+
+private const val TAG = "DefaultWebViewEventListener"
+
+/**
+ * This class is the default implementation of WebViewEventListener.
+ * It can be used with delegation pattern
+ */
+
+class DefaultWebViewEventListener : WebViewEventListener {
```

Comment:
Maybe add `@CallSuper` on each methods?

Reply:
This class is not open and shouldn't be extended, so nope :-)